clients/unshaded/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -143,5 +143,21 @@ public static void accessReinitializer(ReinitializerAccesser accesser) {
     accesser.receiveAccess(new PrivateReinitializer());
   }
 
-  private ClientContext() {}
+  /**
+   * Reset the TachyonConf instance in worker context, for test only.
+   * TODO(binfan): consider a better way to mock test TachyonConf
+   */
+  public static void resetConf() {
+    sTachyonConf = new TachyonConf();
```

Comment:
It seems that it is just to reset the TachyonConf without modifying other fields.